lib/control.rb

```diff
@@ -0,0 +1,112 @@
+class Control
+  require_relative './plateu'
+  require_relative './rover'
+  require_relative './houston_command'
+  attr_accessor :x, :y, :plateu, :rover, :commands, :command_array
+
+  def command_rover(string)
+    @command_array = string
+    set_up_plateu_and_delete
+    set_up_rover_and_send_commands
+     while check_for_commands(@command_array)
+       reset_rover_and_send_commands
+     end
+  end
+
+  def set_up_plateu_and_delete
+    set_up_plateu(@command_array)
+    delete_first_two(@command_array)
```

Comment:
It would help readability for this line to indicate why two things are being deleted.  eg indicate what the commands are that are being deleted.